### DISCUSSION TITLE
Sum of items with problems?

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1937] [Q&A] [answered]
The image below shows the result of a search where we can see in the first line that the total results returned are 1,204 items, 1,079 in the first HD and 125 in the second HD.

In the second line I just turned on the filters flag. And the result was 440 items, 398 in the first HD and 115 in the second HD, but 398+115 = 513 and not 498. I can't even imagine where the difference of 73 came from.
Any light?

![image](https://github.com/sepinf-inc/IPED/assets/381594/7d66b8ac-0829-4824-9b3c-466da2d2d8a4)

**Comment by wladimirleite** [comment]
There are 73 items present in both evidences.

**Reply by wladimirleite** [reply, marked as answer]
In the first evidence there 398 unique items, and in the second, 115.
If some of these items are present in both, when you do not filter a particular evidence, the number of unique items would be lower than the sum.
In theory, It could be some number between 398 and 513, inclusive. In your case, it is 440.

**Reply by paulobreim** [reply]
Great. I hadn't thought of it that way.

tks
